packages/components/src/input/src/components/osds-input/core/controller.ts

```diff
@@ -54,6 +55,9 @@ class OdsInputController {
   onValueChange(value: OdsInputValue, oldValue?: OdsInputValue): void {
     this.logger.debug(`[input=${this.component.value}]`, 'value changed', { oldValue, value });
     this.assertValue(value);
+    if (this.component.inputEl) {
+      this.component.inputEl.value = value?.toString() ?? '';
+    }
     this.component.internals.setFormValue(value?.toString() ?? '');
```

Comment:
as in main PR, carefull with string cast (and on above line)

Reply:
This is "normal" because in a native input, the value is a string